examples/google.events.cloud.audit.v1.md

```diff
@@ -0,0 +1,34 @@
+## google.events.cloud.audit.v1
+
+Example cURL request with headers and data format seen by Cloud Run:
+
+```sh
+curl localhost:8080 -XPOST \
+-H "host: my-run-url-uc.a.run.app" \
+-H "content-type: application/json" \
+-H "accept: application/json" \
+-H "from: noreply@google.com" \
+-H "user-agent: APIs-Google; (+https://developers.google.com/webmasters/APIs-Google.html)" \
+-H "x-cloud-trace-context: 9df407335d114d56ea27ee69ff12943f/17236816549570278484" \
+-H "x-forwarded-for: 66.102.6.152" \
+-H "x-forwarded-proto: https" \
+-H "forwarded: for='66.102.6.152';proto=https" \
+-H "accept-encoding: gzip,deflate,br" \
+-H "ce-id: 1413058901901494" \
+-H "ce-source: //pubsub.googleapis.com/projects/MY-PROJECT/topics/MY-TOPIC" \
+-H "ce-specversion: 1.0" \
+-H "ce-type: com.google.cloud.pubsub.topic.publish" \
+-H "ce-time: 2020-12-08T20:03:19.162Z" \
+--data-binary @- << EOF
+{
+  "data": {
+    "message": {
+      "data": "SGVsbG8gdGhlcmU=", 
```

Comment:
I think it might be useful to have an example that has attributes as well as data.

Reply:
Sounds fine, but I'd rather generate a Pub/Sub message with this at a later point. This is from a real Pub/Sub message and is valid.

Eventually we should have multiple examples like seen with https://github.com/GoogleCloudPlatform/functions-framework-conformance/tree/master/events/generate/data.